apps/studio/pages/project/[ref]/database/security-advisor.tsx

```diff
@@ -31,12 +32,49 @@ const ProjectLints: NextPageWithLayout = () => {
   const [selectedRow, setSelectedRow] = useState<number>()
   const [selectedLint, setSelectedLint] = useState<Lint | null>(null)
 
-  const { data, isLoading, isRefetching, refetch } = useProjectLintsQuery({
+  const {
+    data,
+    isLoading,
+    isRefetching,
+    refetch: refetchLintsQuery,
+  } = useProjectLintsQuery({
     projectRef: project?.ref,
     connectionString: project?.connectionString,
   })
 
-  const activeLints = data?.filter((x) => x.categories.includes('SECURITY')) || []
+  const { data: authConfig, refetch: refetchAuthConfigQuery } = useAuthConfigQuery({
+    projectRef: project?.ref,
+  })
+
+  const refetch = () => {
+    refetchLintsQuery()
+    refetchAuthConfigQuery()
+  }
+
+  let clientLints: Lint[] = []
+
+  // [Alaister]: checking this client side for speed, but should be moved into the query if possible
+  if (authConfig?.EXTERNAL_EMAIL_ENABLED && authConfig.MAILER_OTP_EXP > 3600) {
+    clientLints.push({
+      name: 'auth_otp_long_expiry',
+      level: 'WARN',
+      facing: 'EXTERNAL',
+      categories: ['SECURITY'],
+      description: 'OTP expiry exceeds recommended threshold',
+      detail:
+        'We have detected that you have enabled the email provider with the OTP expiry set to more than an hour. It is recommended to set this value to less than an hour.',
```

Comment:
```suggestion
        'We have detected that you have enabled the email provider with the OTP expiry set to more than an hour. It is recommended to set this value to less than an hour to prevent brute force attacks.',
```